server/schedule/placement/fit.go

```diff
@@ -398,6 +380,28 @@ func (p *fitPeer) matchRoleStrict(role PeerRoleType) bool {
 	return false
 }
 
+func isolationStoreScore(srcStoreID uint64, dstStore *core.StoreInfo, stores []*core.StoreInfo, labels []string) float64 {
+	var score float64
+	if len(labels) == 0 || len(stores) <= 1 {
+		return 0
+	}
+	for i := range stores {
+		store1 := stores[i]
+		if store1.GetID() == srcStoreID {
+			store1 = dstStore
```

Comment:
why set it to dstStore

Reply:
the current isolated score conside all the stores like [1,2,3], and the scheduler isolated score conside the new stores like [1,2,4], so the store-3 will be replace by store-4